pkg/access_token/access_token.go

```diff
@@ -0,0 +1,17 @@
+package access_token
+
+import (
+	"fmt"
+
+	sdk "github.com/openshift-online/ocm-sdk-go"
+)
+
+func CreateAccessToken(conn *sdk.Connection) error {
+
```

Comment:
nitpick: we don't need the line break here. Let's remove it to make it consistent with the rest of the code.